### DISCUSSION TITLE
[Features] Stage ID calling from char, Partial Reflections

### DISCUSSION THREAD

**Discussion by Amidweiz** [ikemen-engine/Ikemen-GO#839] [Feature Request] [answered]
One thing I've noticed with the StageVar trigger is that it is only limited to getting info about the author, displayname, and name. It would be nice if one could detect when a ID within a stage def is existent or not which could help with implementing a true stage intro with any character having to wait for it to end with the stagechar.

Another thing that could be implemented is for specific elements in a stage to reflect rather than just the whole stage to reflect back the image of the player which is how it's done within mugen. With bg sctrls having a `reflect = 1` code in them which it would use the stages reflect values as default and maybe having their how set of reflect values if one desired.

**Comment by K4thos** [comment]
what do you mean by ID in this context? Are you referring to ID number for a background element? I don't understand how that could be useful for stage intro.

**Comment by K4thos** [comment, marked as answer]
detecting stage by id that can be shared by multiple stages is already implemented:
https://github.com/K4thos/Ikemen_GO/wiki/Stage-features#constants
